ext/cron/01_cron.ts

```diff
@@ -103,10 +105,14 @@ function cron(
   handler2?: () => Promise<void> | void,
 ) {
   if (name === undefined) {
-    throw new TypeError("Deno.cron requires a unique name");
+    throw new TypeError(
+      "Cannot create cron job, a unique name is required: received `undefined`",
```

Comment:
```suggestion
      "Cannot create cron job, a unique name is required: received 'undefined'",
```

Reply:
Good catch. I'm going to try and use the single ticks everywhere. Maybe we should add that to the style guide too.